transliteration/src/test/java/edu/illinois/cs/cogcomp/transliteration/TransliterationAnnotatorTest.java

```diff
@@ -22,14 +23,13 @@
     TransliterationAnnotator annotator = null;
     @Before
     public void setUp() throws Exception {
-        this.annotator = new TransliterationAnnotator();
+        this.annotator = new TransliterationAnnotator(true, Language.Persian);
     }
 
-/*    @Test
+    @Test
     public void testCleanText() throws AnnotatorException {
         TextAnnotation ta = DummyTextAnnotationGenerator.generateAnnotatedTextAnnotation(false, 3);
         annotator.getView(ta);
         assertEquals(true, ta.hasView(ViewNames.TRANSLITERATION));
```

Comment:
this is a bit minimal -- can you test that there is some expected transliteration of a word in 'ta'?

Reply:
Done.